### ISSUE TITLE
Example description has something wrong

### ISSUE THREAD

**Issue by songbongkeun** [raywenderlich/swift-style-guide#290]
https://github.com/raywenderlich/swift-style-guide#minimal-imports

it says 
```Keep imports minimal. For example, don't import _UIKit_ when importing _Foundation_ will suffice.```

should be change
```Keep imports minimal. For example, don't import  **Foundation** when importing **UIKit** will suffice.```

**Comment by hollance** [comment]
Both are correct. I think the style guide currently means that if you don't need UIKit, then don't import it. What you are saying is, if you already import UIKit then don't also import Foundation.

**Comment by songbongkeun** [comment]
@hollance I didn't understand style guide says `if you don't need UIKit, then don't import it`. I understand style guide says '_Foundation_ include _UIKit_ so do not import _UIKit_ althought you need. It will import automatically by _UIKit_`
If my understand not normal, please ignore or close this issue. Thanks.

**Comment by JessyCatterwaul** [comment]
@songbongkeun I believe the problem is that `minimal` is not technical jargon. 

For you, it meant:
 "Do not import **two** modules, `Module U` and `Module F`, when you're already importing `Module U`."

To some of the rest of us, it means: 
"Do not import `Module U`, which imports `Module F` when `Module F` will suffice".
I'm saying that the word `minimal` is not universal for this esoteric concept. But I also don't know if there is more accurate common terminology. `Minimal,` here, refers to the scope of sets (`Module U` becomes a superset of `Module F`), not to the number of `import` statements.

I don't think the guideline against what you're thinking is too helpful, because it doesn't come up when coding. I do think the existing guideline is useful, but if it could be expressed more clearly for you, please make a suggestion as to what would do that. If it confused you, some other people are going to be confused by it as well, and they might not speak up.

**Comment by songbongkeun** [comment]
@JessyCatterwaul Right. Word `minimal` is not clear. I don't know owner consider `indirectly import` but I think `minimal` should include that(It can be only my opinion..). How about below change?


### Minimal Imports
Keep imports minimal. 
* Don't import UIKit when importing Foundation will suffice.
* Don't duplicate import consider indirectly import.

**Preferred:**
```swift
import UIKit
var view: UIVIew
var deviceModels: Array<String>
```

**Not Preferred:**
```swift
import UIKit
import Foundation
var view: UIVIew
var deviceModels: Array<String>
```
```swift
import UIKit
var deviceModels: Array<String>
```
